Makefile

```diff
@@ -10,29 +10,26 @@ clean-ctags:
 clean: clean-ctags
 	rm -rf .pytest_cache
 	find . -type f -name '*.pyc' | xargs rm -f
-    find . -type d -name '__pycache__' | xargs rm -f
+	find . -type d -name '__pycache__' | xargs rm -rf
 
 install:
-    cd databoard && pip install . && cd ..
-    cd ramp-database && pip install . && cd ..
-    cd ramp-engine && pip install . && cd ..
+	cd databoard && pip install . && cd ..
+	cd ramp-database && pip install . && cd ..
+	cd ramp-engine && pip install . && cd ..
 
-test-all:
-    pytest -vsl
-
-test: test-all
+test: test-db test-frontend test-engine
```

Comment:
can't you keep this as `pytest` since you defined the `testpaths` in the ini file?

Reply:
I tried. But it actually throws an error that I cannot understand
```
=================================================================================================================== ERRORS ===================================================================================================================
_________________________________________________________________________________________ ERROR collecting ramp-database/rampdb/tests/test_utils.py __________________________________________________________________________________________
import file mismatch:
imported module 'rampdb.tests.test_utils' has this __file__ attribute:
  /Users/aboucaud/codes/python_venvs/rampwf3/lib/python3.6/site-packages/rampdb/tests/test_utils.py
which is not the same as the test file we want to collect:
  /Users/aboucaud/work/RAMP/ramp-board/ramp-database/rampdb/tests/test_utils.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```

I cleaned the repo as they say but it did not change a thing. Can you try ?

Reply:
for me it is working, but the default of is very noisy (all the logging info)